bin/wasabi.sh

```diff
@@ -347,26 +353,21 @@ for command in ${@:$OPTIND}; do
     bootstrap) bootstrap;;
     build) build true;;
     clean) clean;;
-    start) command="start:cassandra,mysql,wasabi";&
-    start:*) commands=$(echo ${command} | cut -d ':' -f 2)
-      (IFS=','; for command in ${commands}; do start ${command}; done);;
+    start) exec_commands start "cassandra,mysql,wasabi";;
+    start:*) exec_commands start ${command};;
     test:*) commands=$(echo ${command} | cut -d ':' -f 2)
       (IFS=','; for command in ${commands}; do mvn "-Dtest=com.intuit.wasabi.${command/-/}.**" test -pl modules/${command} --also-make -DfailIfNoTests=false -q ; done);;
```

Comment:
That was good idea, I created a new unit_test() function and called it in a similar way as rest of the options/commands... Please refer to the latest commit.